pkg/rbd/rbd_attach.go

```diff
@@ -193,6 +194,7 @@ func attachRBDImage(volOptions *rbdVolume, cr *util.Credentials) (string, error)
 }
 
 func createPath(volOpt *rbdVolume, cr *util.Credentials) (string, error) {
+	var isNbd bool
```

Comment:
I prefer explicit assigning of variables, like `isNbd := false` in this case

Reply:
done.

Reply:
@nixpanic The default of `bool` is `false`.  iic, the explicit assigning is not the recommended practice if its on default value..

Reply:
Sure, a bool defaults to false, but having it stated makes the code more readable (at least to me). The main objection was not for this bool, but for the not logging of the original error.